src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
 spring.application.name=GitHubAiAPI
+
+
```

Comment:
The property 'Spring.demo.value' should follow the standard naming convention. It should be in lowercase and separated by periods. Consider renaming it to 'spring.demo.value'.